### ISSUE TITLE
Thunderbird 102 and newer

### ISSUE THREAD

**Issue by lluisalemanypuig** [protz/LatexIt#93]
Hello! This issue is more of a question than a bug report, or otherwise.

I discovered this addon back when it was available for Thunderbird 70 and something (I think, about two years ago -- I really don't remember exactly). I really liked it and used it to write emails with math in them for as long as Thunderbird allowed me to. I mean, I had to update Thunderbird at some point, and after that the plugin wasn't compatible with Thunderbird anymore. At the time of writing, I'm using Thunderbird 102.2.2.

I was wondering if you're planning to update the plugin so that it's compatible with newer versions of Thunderbird.

If you did so, I would very much appreciate it!

**Comment by nielsbonn** [comment]
It's 2025 and this still seems to be the only add-on with this functionality. Would really appreciate if this could come back compatible with current versions of Thunderbird. Anyone around who has the knowledge to do this?

**Comment by sphh** [comment]
Yes, that would be brilliant! I might have done some maintenance work on the add-on, but that is how much my knowledge goes. I also got the feeling, that the original author has lost interest.

What about you, @nielsbonn?

**Comment by protz** [comment]
Well it's tough because I don't use this addon anymore myself, so the incentives for me to keep it up to date are nonexistent.

@sphh do you have a sense of how much work is needed for this upgrade?

**Comment by sphh** [comment]
Absolutely no idea. And it is way above my paygrade (as they say): JavaScript is _not_ ‘my’ language; I am just able to fiddle with existing code, but writing completely new JavaScript code is out of my reach.

BTW Thunderbird supports formulae in MathML (open a HTML mail and go to `Insert > Math`). No idea which other mail clients support displaying MathML …

**Comment by nielsbonn** [comment]
> Yes, that would be brilliant! I might have done some maintenance work on the add-on, but that is how much my knowledge goes. I also got the feeling, that the original author has lost interest.
> 
> What about you, [@nielsbonn](https://github.com/nielsbonn)?

Sorry, I don't really have any knowledge to work on this. I'm just a user.

It's true, the MathML option is there. Maybe we should just go for that? But I liked LatexIt because it was very keyboard friendly (just type $ or $$), instead of clicking "Insert -> Math" for every tiny formula. And of course, one knew that the recipient would be able to read it as long he was able to read any HTML mail...

**Comment by nielsbonn** [comment]
Addition: I just checked with a colleague. For him, MathML formulas arrive as an unreadable mix of symbols and tex code, without spacing or line breaks. Unusable.

It would really be nice to get back something like LatexIt which produced a reliable result.
